sentences/pl/homeassistant_HassGetState.yaml

```diff
@@ -18,6 +18,14 @@ intents:
             - scene
             - script
 
+      - sentences:
+          - stan <name> [<area>]
+        response: one_state
+        excludes_context:
+          domain:
+            - scene
+            - script
+
       - sentences:
           - jak(i|a|e) jest status <name> [<area>]
```

Comment:
Can we please also fix the `jake` issue I can see here, please adapt this to `(i|a|ie)`

Reply:
also removed jaka since noone talks like that